Dockerfile

```diff
@@ -5,7 +5,7 @@ WORKDIR /usr/src/app
 
 COPY package.json /usr/src/app
 COPY package-lock.json /usr/src/app
-RUN npm install
+RUN npm install --production
```

Comment:
why restrict docker to production only?

Reply:
@shay123g

Well, you're not developing inside it, so you don't need eslint etc.